features/video-transformation/resize-crop-and-other-common-video-transformations.md

```diff
@@ -290,16 +290,66 @@ URL - [https://ik.imagekit.io/demo/sample-video.mp4?tr=n-video\_thumbnail](https
 
 ### Background color - (bg)
 
-It is used to specify the background color in RGB Hex Code (e.g. FF0000). This is usually used with [pad\_resize](resize-crop-and-other-common-video-transformations.md#pad-resize-crop-strategy-cm-pad\_resize) cropping to control the color of extra background padding.
+It is used to specify the background for [crop-mode pad\_resize](resize-crop-and-other-common-video-transformations.md#pad-resize-crop-strategy-cm-pad\_resize) or border-radius (r). Background can be any solid color or blurred frame of the image or video.
```

Comment:
It is used to specify the background of the video when using pad_resize or border-radius. The background can be any solid colour or blurred frame of the video.

images is not needed here.